Add comment about uenvs (#303)
C2SM/c2sm.github.io: docs/models/icon/compile_and_run.md

!!! Note

>     If you have never used an uenv von Säntis, you need to create a uenv repo first: `uenv repo create`.

Reviewer: ```suggestion
    If you have never used a uenv on Säntis, you need to create a uenv repo first: `uenv repo create`.
```
